CSS/style.css

```diff
@@ -11,6 +11,71 @@ body {
     background-color: #bfbfbf;
 }
 
```

Comment:
`*{
     font-family: 'Bree Serif', serif;
}`

You can replace all the individual uses of `font-family: 'Bree Serif', serif;` with that^